litmus-portal/frontend/src/components/Sections/Workflow/ChooseWorkflow/index.tsx

```diff
@@ -102,12 +103,17 @@ const ChooseWorkflow: React.FC = () => {
       id: workflowsList[index].workflowID,
       yaml: 'none',
       description: workflowsList[index].description,
-      isCustomWorkflow: false,
+      isCustomWorkflow: workflowsList[index].isCustom,
     });
+
     setWorkflowData({
       workflowName: workflowsList[index].title,
       workflowDesc: workflowsList[index].description,
     });
+
+    if (workflowsList[index].isCustom === true) {
+      setOpen(true);
+    }
```

Comment:
In which case this would never happen because there are no custom workflow card in this release

Reply:
I'm working on custom workflow screens flow as well so that will be needed there.
if not implemented in backend, i will just disable the custom workflow.